torch/_inductor/ir.py

```diff
@@ -5430,7 +5428,21 @@ def require_channels_last_3d(cls, x):  # type: ignore[no-untyped-def]
 
     @classmethod
     def require_contiguous(cls, x):  # type: ignore[no-untyped-def]
-        return cls.require_stride_order(x, list(reversed(range(len(x.get_size())))))
+        def safe_get_name(x):
+            try:
+                return x.get_name()
+            except (AttributeError, NotImplementedError):
+                return None
+
+        if (
```

Comment:
Nit: can you factor this out to a helper method ?

Reply:
Factor this check to the method of `is_mkldnn_tensor`.